hotsos/core/host_helpers/cli.py

```diff
@@ -1,127 +1,32 @@
 import abc
 import datetime
-import glob
 import json
 import os
 import pathlib
-import pickle
 import re
 import subprocess
 import tempfile
 from functools import cached_property
 
 from hotsos.core.config import HotSOSConfig
-from hotsos.core.host_helpers.common import HostHelpersBase
+from hotsos.core.host_helpers.common import (
+    CmdOutput,
+    get_ps_axo_flags_available,
+    HostHelpersBase,
+    reset_command,
+    run_post_exec_hooks,
+    run_pre_exec_hooks,
+    SourceRunner,
```

Comment:
I feel like these are not general enough to belong in host_helpers.common and would be better placed in e.g. host_helpers.cli_common.py